indico/modules/events/abstracts/models/reviews.py

```diff
@@ -137,3 +139,9 @@ class AbstractReview(db.Model):
     @return_ascii
     def __repr__(self):
         return format_repr(self, 'id', 'abstract_id', 'user_id', proposed_action=None)
+
+    @property
+    def score(self):
+        if not self.ratings:
+            return None
+        return sum((x.value for x in self.ratings)) / len(self.ratings)
```

Comment:
I was assuming we store the value within the scale. If that's not the case we have to adapt this, for sure.